mu-plugins/blocks/global-header-footer/postcss/footer/logos.pcss

```diff
@@ -115,13 +115,17 @@
 		margin-bottom: 0;
 		font-family: serif; /* MrsEaves doesn't support non-latin characters well. */
 		font-size: 20px;
-		color: #4f5357;
+		color: rgba(255, 255, 255, 0.25);
 		text-transform: uppercase;
 		letter-spacing: 0.1em;
 
 		@media (--tablet) {
 			margin-top: 0;
 			text-align: center;
 		}
+
+		.is-dark {
```

Comment:
I can't remember if we prefer chaining short classes or extending `{component}--is-dark`. I couldn't find any examples and the block editor [coding guidelines](https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/) don't appear to be clear.

Reply:
Short for these `is-`/`has-` style classes 👍🏻 

But you don't need to create a new class, you could change the color based on the background color:

```css
.wp-block-group.global-footer {
	&.has-white-background-color span.global-footer__code_is_poetry {
		color: #8f8f8f;
	}
}
```

If you do want to stick with `is-dark`, this needs the `&` prefix, otherwise the styles don't build correctly.

<img width="273" alt="Screen Shot 2022-08-25 at 12 46 36 PM" src="https://user-images.githubusercontent.com/541093/186723317-f4569fb9-3a07-4875-9685-03cdddf08e46.png">

Reply:
> But you don't need to create a new class, you could change the color based on the background color:

Right, I mistakenly thought these were applied by Gutenberg.  👍